packages/documentation-site/patternfly-docs/content/components/pagination/pagination.md

```diff
@@ -6,65 +6,84 @@ section: components
 
-#### Mobile full pagination in toolbar
+### Compact pagination
 
 <div class="ws-docs-content-img">
-![Example of pagination on mobile table view.](./img/mobile-pagination.svg)
+![Example of a compact pagination.](./img/compact-pagination.svg)
 </div>
 
-1. **Items in view:** At this screen size, the top toolbar only shows the item count to indicate items currently in view. It hides all other pagination controls.
+1. **Per-page selector:** Used to choose the number of items displayed on a single page at once.
+2. **Previous page/next page:** Used to navigate back or forward one page of content at a time.
 
-2. **Full pager:** Supplies the same pagination options as the desktop full pager, but in the bottom toolbar.
+Compact pagination reduces visual crowding and saves space for additional actions in limited spaces, like toolbars. 
 
-### Compact pagination
+### Indeterminate pagination
 
-Compact pagination fits more actions into top toolbars with limited space. Choose this compact variation when your top toolbar contains many items or looks overcrowded. As with full pagination, the compact variation condenses to an item count for mobile views.
+When it's difficult to calculate the total number of items or page count upfront, you can use an indeterminate pagination, which replaces the total page number in the per-page selector with a general label of “many”. Users can still choose the number of items shown at once, and the indeterminate label can be used within both full and compact pagination.
 
-#### Desktop compact pagination in toolbar
+<div class="ws-docs-content-img">
+![Example of an indeterminate pagination.](./img/indeterminate-pagination.svg)
+</div>
+
+## Placement
+
+### Desktop
+
+For desktop views, place [full pagination](#full-pagination) above the content it controls, typically within a toolbar.
 
 <div class="ws-docs-content-img">
-![Example of a compact pagination.](./img/compact-pagination.svg)
+![an example of a desktop pagination component within a toolbar.](./img/toolbar-pagination.svg)
 </div>
 
-1. **Items in view:** Allows the user to select the item count (number of listed items) per page, as seen in full pagination.
+When used in a toolbar, pagination may leave limited room for other items like filters or bulk selectors. These additional items should automatically move into an [overflow menu](/components/toolbar/design-guidelines#overflow-menu) as needed.
 
-2. **Compact pager:** Supplies the user with page-back and page-next controls only.
+For components with long, scrolling content (like a table), you can add a second full pagination component below the content to ensure that users can navigate without scrolling back to the top.
```

Comment:
Just my take on it but since the top full pagination turns into a summary on mobile, I would expect bottom pagination to be used by default, unless you can't - otherwise you lose pagination on mobile.

And I think it's worth calling out bottom pagination as "bottom" instead of (or along with) "second full pagination below" since technically "bottom" is a variant of the pagination component (`<Pagination variant="bottom">`), just so it's clear. As a designer, it probably doesn't matter to call out the distinction since they look the same, but as a dev `variant="bottom"` is what applies the unique sticky layout on mobile.